.taskcluster.yml

```diff
@@ -168,11 +168,21 @@ tasks:
                                         # staging release promotion on forks.
                                         - $if: 'tasks_for == "github-push"'
                                           then:
+                                              - index.mobile.v2.${project}.branch.${short_head_branch}.latest.taskgraph.decision
+                                              - index.mobile.v2.${project}.branch.${short_head_branch}.revision.${head_sha}.taskgraph.decision
+                                              - index.mobile.v2.${project}.revision.${head_sha}.taskgraph.decision
```

Comment:
I'm going to move the `mobile.v2.fenix.revision.*` scopes to [bug 1631834](https://bugzilla.mozilla.org/show_bug.cgi?id=1631834) for 2 reasons:

1. These scopes don't have their counterpart in `project.mobile.fenix`
2. I need to grant them. As of now, only `mobile.v2.fenix.branch.*` is allowed.